hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -210,6 +401,10 @@ canCloseWithLongContestationPeriod tracer workDir node@RunningNode{networkId} hy
     (fuelUTxO, otherUTxO) <- queryMarkedUTxO node actorVk
     traceWith tracer RemainingFunds{actor = actorName actor, fuelUTxO, otherUTxO}
 
+mkDraftUTxOs :: UTxO -> Maybe ScriptInfo -> UTxO' DraftUTxO
```

Comment:
This is a weird signature and it's semantics will pair the script info to each entry of the `UTxO`, which could be surprising.

Could: Make this a proper smart constructor with signature `mkDraftUTxO :: TxIn -> TxOut -> Maybe ScriptInfo -> UTxO' DraftUTxO` or so.

If this should be hard to use because the `createOutputAtAddress` returns `UTxO`, change that function to return `(TxIn, TxOut)` and do `mkDraftUTxO :: (TxIn, TxOut, Maybe ScriptInfo) -> UTxO' DraftUTxO` instead.